website/source/layouts/blog-post.html.haml

```diff
@@ -0,0 +1,18 @@
+!!!html5
+%html
+  = partial "layouts/head"
+  %body.layout(class="#{"-subpage-" unless data.page.root}")
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))